compliance_checker/cf/cf.py

```diff
@@ -125,7 +126,13 @@ def _find_cf_standard_name_table(self, ds):
         # Try to parse this attribute to get version
         version = None
         if 'cf standard name table' in standard_name_vocabulary.lower():
-            version = standard_name_vocabulary.split()[-1]
+            version = [s.strip('(').strip(')').strip('v').strip(',') for s in standard_name_vocabulary.split()]
+            # This assumes that table version number won't start with 0.
+            version = [s for s in version if s.isdigit() and len(s) <= 2 and not s.startswith('0')]
+            if len(version) > 1:
+                return False
+            else:
+                version = version[0]
```

Comment:
@Bobfrat do you want me to do something here? Maybe re-factor this in 3 PRs?
We have here:

- AppVeyor updates;
- "improved" parsing of the `standard_name_vocabulary` version;
- issue a warning instead of a print on the screen when failing to get the version.

Reply:
Nah sorry, this is ok to me, I'll merge